src/maybe.ts

```diff
@@ -285,4 +293,8 @@ export class Nothing<T = unknown> {
     public toResult<E>(error: E): Result<E, T> {
         return Result.err(error)
     }
+
+    public toUnion<O>(other: O): T | O {
```

Comment:
Personally, I would just return `T | null` here instead of forcing a default. I suppose that you could just do `maybe.toUnion(null)` if you really wanted to though

Reply:
That make sense, but I want to use in a project without strict null checks, so `T | null` will just become `T` quietly.